run-faster-incremental-java.sh

```diff
@@ -1,9 +1,14 @@
-#!/bin/sh
+#!/bin/bash
+
+AGP_VERSION="${AGP_VERSION:-3.6.1}"
+GRADLE_VERSION="${GRADLE_VERSION:-6.2.2}"
+
 sync
 
 gradle-profiler/gradlew --project-dir gradle-profiler install
 
 gradle-profiler/build/install/gradle-profiler/bin/gradle-profiler \
+    -DagpVersion=$AGP_VERSION \
```

Comment:
Why are you passing this system property to Gradle profiler? Don't you need to pass it to the santa tracker build?

Reply:
It passes them on as expected, apparently. How would I only pass it to the builds themselves?

Reply:
I would have added something to the scenario file. I am actually surprised Gradle profiler passes system properties from the command line to the Gradle builds it runs. Is that documented somewhere?

Reply:
I don't know how to do this in a nice way in HOCON. It works now with the bash script, so I'd leave it as is.

I'd argue that the scenario file should handle things _inside_ Gradle, while the Bash scripts should handle the environment.

Reply:
You can access environment variables in HOCON: https://github.com/lightbend/config/blob/master/HOCON.md#substitution-fallback-to-environment-variables

It seems to me that the `agpVersion` should be part of the scenario file, then we would at least see in the gradle profiler log which AGP version is used, since it would be a parameter to the Gradle invocation.

Your argument is that the bash script should track the Gradle and the AGP versions? I can live with that as well.

Reply:
That works the other way around: you can use the environment as a fallback. What I'd need is to take the environment variable's value, and fall back to some other value if not specified.

Anyway, let's leave it like this for now, and change it if it causes problems.